### DISCUSSION TITLE
Multiple SQL query in a single task

### DISCUSSION THREAD

**Discussion by michelevioglio97** [elsa-workflows/elsa-core#4023] [Q&A]
In order to retrieve more variables to be saved in the context from different tables and keep the queries simple, i would like to write more SELECT commands in a single task. Do you have an idea if it's already possible to do so?
Example:
Task SQL Query:
SELECT a,b
from tab1
SELECT c,d
from tab2

and so on, in a unique task

**Comment by sfmskywalker** [comment]
Makes sense. I don't think this is currently possible, but would be interesting to consider when porting the SQL activities to V3. Perhaps you want to create an issue for it?
